src/hotspot/os/bsd/os_bsd.cpp

```diff
@@ -1560,7 +1560,13 @@ void os::print_os_info(outputStream* st) {
 }
 
 void os::pd_print_cpu_info(outputStream* st, char* buf, size_t buflen) {
-  // Nothing to do for now.
+  char model[100];
```

Comment:
You don't need to declare a buffer to store the model name, the caller passes one in.